bpf/net/bpf_filter.c

```diff
@@ -618,7 +618,7 @@ bpf_filter(pc, p, wirelen, buflen)
 int
 bpf_validate(f, len)
 	const struct bpf_insn *f;
-	int len;
+	u_int len;
```

Comment:
OK. I removed that modification at: https://github.com/the-tcpdump-group/libpcap/pull/512/commits/c2d601298b8f57a18fe21ae735360d92e5ebc487